binding/android/Leopard/leopard/src/main/java/ai/picovoice/leopard/Leopard.java

```diff
@@ -211,6 +217,16 @@ public Builder setEnableAutomaticPunctuation(boolean enableAutomaticPunctuation)
             return this;
         }
 
+        /**
+         * Setter for enabling speaker diarization.
+         *
+         * @param enableDiarization Set to `true` to enable speaker diarization.
```

Comment:
add the full description that you used in the constructor, so that the user can understand what it is from the builder as well